example/lib/main.dart

```diff
@@ -11,12 +11,16 @@ import 'package:provider/provider.dart';
 
 import 'theme.dart';
 
+enum MenuSelection {
+  about,
```

Comment:
This is unused as far as I can see?

Reply:
Ah yea, you're correct. I'll remove this when I'm back at my computer, unless you prefer to do it sooner.